content/en/docs/userguide/try_request_routing.md

```diff
@@ -0,0 +1,173 @@
+---
+draft: false
+linktitle: Try Request Routing
+menu:
+  docs:
+    parent: user guide
+    weight: 5
+title: Try Request Routing
+toc: true
+type: docs
+
+
+---
+
+### Preparation
+
+Refer [Install Waypoint | Kmesh](https://kmesh.net/en/docs/userguide/install_waypoint/#preparation)
+
+### Apply version1 routing
+
+1. Run the following command to create the route rules:
+
+```bash
+kubectl apply -f -<<EOF
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: reviews
+spec:
+  hosts:
+    - reviews
+  http:
+  - route:
+    - destination:
+        host: reviews
+        subset: v1
+      weight: 100
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: reviews
+spec:
+  host: reviews
+  trafficPolicy:
+    loadBalancer:
+      simple: RANDOM
+  subsets:
+  - name: v1
+    labels:
+      version: v1
+  - name: v2
+    labels:
+      version: v2
+  - name: v3
+    labels:
+      version: v3
+EOF
+```
+
+2. Display the defined routes with the following command:
+
+```bash
+kubectl get virtualservices -o yaml
+```
+
+You have configured Kmesh to route to the `v1` version of the Bookinfo microservices, most importantly the `reviews` service version 1.
```

Comment:
bookinfo is a collection of all applictions of `productpage`, `reviews`, `details` and `ratings`.